### PR TITLE
Complete overhaul of entry page.

### PR DESCRIPTION
**Pull request by natsteinmetz**
This PR contains:
* responsive components for site-sections; styles for main & sidebar areas
* main sections for resources, community, and mentorship
* sidebar sections for purpose and what we're working on
* as-seen-on video iframe section
* recoloring of navbar and footer

### CODE REVIEW

**Comment by cherchezlafemme** [comment]
Looking at PR now, will fix stuff as I see it to speed up the process.

**Comment by natsteinmetz** [comment]
Awesome, thanks, Anya!

**Comment by cherchezlafemme** [comment]
@natsteinmetz do we want add footer to entry page? it's present on other pages. We also can update the footer to be 2015-2018

**Comment by cherchezlafemme** [comment]
there is bunch of duplicate css, and mobile doesn't look good. working on rewriting it.

**Comment by cherchezlafemme** [comment]
some bunch of duplicate css, mobile doesn't look good. working on updating it.

**Comment by jadefaerie** [comment]
I was just wondering if the footer was added to the front page - I can't check right now, but I am hoping that change was included? If not, I agree we should :)

**Comment by natsteinmetz** [comment]
I agree, we want the footer added and changed. But we can do this in a separate branch/PR too. If all looks good we can deploy this very soon! :)

File: app/views/static_pages/front_page.html.erb [hunk 1]
```diff
@@ -1,38 +1,139 @@
...
+              <div class="iframe-container">
+                <iframe src="https://player.vimeo.com/video/136884902" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+              </div>
+            </section>
         </div>
     </div>
-</div>
+  </main>
+
+  <%= render :partial => "layouts/footer" %>
```
Comment (cherchezlafemme) [inline app/views/static_pages/front_page.html.erb]: I added the footer in this PR. You should see it if pull changes and run locally.

File: app/views/layouts/_footer.html.erb [hunk 2]
```diff
@@ -2,12 +2,14 @@
   <div class="container-fluid">
     <div class="clearfix">
       <ul class="ui-ul list-inline pull-left">
-        <li>&copy; 2015 JOLT Labs LLC and individual contributors
-        <li> &nbsp;&nbsp;&nbsp; | &nbsp;&nbsp;&nbsp;</li>
-        <li><%= mail_to 'info@shescoding.org','info@shescoding.org' %></li>
+        <li>
+          &copy; 2015 - <%= Time.current.year %> JOLT Labs LLC and individual contributors
```
Comment (cherchezlafemme) [inline app/views/layouts/_footer.html.erb]: Yep, I added time.current.year so it's always up to date :D

**Comment by cherchezlafemme** [comment]
Only thing this PR is missing is the background color of different sections. Didn't get to it. :/

File: app/views/static_pages/front_page.html.erb (hunk 1 above)
Comment (natsteinmetz) [reply app/views/static_pages/front_page.html.erb]: Yup, I saw it - thanks Anya! :D

**Comment by natsteinmetz** [comment]
Awesome! Yeah, we can just add those later, they're nice to have, but not necessary.

File: app/views/layouts/_footer.html.erb (hunk 2 above)
Comment (natsteinmetz) [reply app/views/layouts/_footer.html.erb]: Perfect.

**Pull request merged**